modules/client/src/controllers/ConditionalTransferController.ts

```diff
@@ -0,0 +1,155 @@
+import { DEFAULT_APP_TIMEOUT, HASHLOCK_TRANSFER_STATE_TIMEOUT } from "@connext/apps";
+import {
+  ConditionalTransferTypes,
+  EventNames,
+  HashLockTransferAppName,
+  MethodParams,
+  PublicParams,
+  PublicResults,
+  GenericConditionalTransferAppState,
+} from "@connext/types";
+import { toBN, stringify } from "@connext/utils";
+import { HashZero, Zero } from "ethers/constants";
+
+import { AbstractController } from "./AbstractController";
+import { soliditySha256 } from "ethers/utils";
+
+export class ConditionalTransferController extends AbstractController {
```

Comment:
Calling this a "ConditionalTransfer" Controller suggests that it's distinct from other types of transfer controllers. But, now that we've consolidated all transfers into one controller, the "Conditional" prefix isn't useful for distinguishing this type of transfer logic from others. I'd recommend calling this module "CreateTransferController" & it's sibling "ResolveTransferController" instead.

Reply:
I agree with this generally but perhaps we should still have an atomic `transfer` method? We'd still need at least one type of transfer to be the "easy"/default transfer that requires minimal overhead.